packages/codec/package.json

```diff
@@ -30,7 +30,8 @@
     "release": "npm publish"
   },
   "dependencies": {
-    "@ckb-lumos/bi": "^0.19.0-alpha.1"
+    "@ckb-lumos/bi": "^0.19.0-alpha.1",
```

Comment:
```suggestion
    "@ckb-lumos/bi": "^0.19.0-alpha.3",
```

Reply:
It will be fixed when merge develop into this PR